### DISCUSSION TITLE
App mounting

### DISCUSSION THREAD

**Discussion by daleif** [mojolicious/mojo#2037] [General]
Been playing a bit with mounting an existing app as a prefix under an existing app.

Am I right in assuming that the mounted app knowns noting about the app it is mounted under? For example session data etc. It is simply doing its own thing under the prefix given.

**Comment by kraih** [comment]
Sessions are merged if you keep using the default cookie name.

**Reply by daleif** [reply]
ohh, that helped, been having a tendency to rename the cookie name.

Now I just need to secure the prefix route, but that is a different matter.
